### ISSUE TITLE
Updates to the Landing Page

### ISSUE THREAD

**Issue by Janetbananet** [responsible-ai-collaborative/aiid#1491]
**Outline:**
Final order of horizontal sections should be as follows:
1. Welcome… + search bar - CHANGES BELOW
2. Latest Incident Report - content the same
3. Quick Add New Report URL - content the same
4. About the Database | Latest Blog Post - CHANGES BELOW
5. Incident Report Submission Leaderboards - content the same
6. Common Entities - content the same
7. Sponsors - CHANGES BELOW

Sections eliminated: The Database in Print, Wordcounts, Random Incidents, The Responsible AI Collaborative

**Content Changes:**
1. Welcome…
Add slogan “Learning from the past to engineer a better future”

4. About the Database | Latest Blog Post: 
About the Database box: Replace with the language with the content below:
_“The AI Incident Database is dedicated to identifying, cataloging, and sharing artificial intelligence incidents with the public. You are invited to [submit](https://incidentdatabase.ai/apps/submit) incident reports, whereupon submissions will be indexed and made [discoverable](https://incidentdatabase.ai/apps/discover) to the world. 

The AI Incident Database is a project of the Responsible AI Collaborative. [(Learn More)](https://incidentdatabase.ai/about)”_

Latest Blog Post box: Change the design so that it looks like it’s part of the “About the Database” section and that it’s more clearly a blog post vs an incident. Here’s a suggestion from a non-UX person :)
Wrap this content into the same box as “About the Database” so that it is still positioned in the right column, but not its own box. Add a title to the column “Latest Blog Posts” and then add smaller thumbnails containing 2 or 3 blog posts (number dependent on how it aligns with “about the DB” content in the left column).  

7. Sponsors: Add this section as a horizontal row across the bottom of the landing page (above the footer). Title of the section should be “Sponsors” and include each of the thumbnails that are currently sitting in “The Responsible AI Collaborative” section.

**Comment by Janetbananet** [comment]
Holding on review and implementation of these changes until January.

**Comment by Janet-ResponsibleAI** [comment]
@datherton09 I don't think this is a high priority right now, but this issue is a good placeholder for future improvements to the landing page.

**Comment by datherton09** [comment]
@Janet-ResponsibleAI Understood! Thank you, Janet!
